src/jit/legacyjit/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ endif(WIN32)
 add_library_clr(legacyjit
    SHARED
    ${SHARED_LIB_SOURCES}
+   ${JIT_ARCH_SOURCES}
```

Comment:
We probably want to define `JIT_ARCH_LEGACY_SOURCES` in `jit/CMakeLists.txt` and use that here. As it stands I think the legacyjit build is going to use the RyuJIT backend.